src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -37,8 +37,9 @@
     // WHen DUMP_TO_FILE is true, if a model or report is dumped, it goes to a file in target/test-dump-output
     private static final boolean DUMP_TO_FILE = false;
 
-    private static final String LDC_NS = "https://tac.nist.gov/tracks/SM-KBP/2018/LdcAnnotations#";
-    private static final String ONTOLOGY_NS = "https://tac.nist.gov/tracks/SM-KBP/2018/ontologies/SeedlingOntology#";
+    private static final String NIST_ROOT = "https://tac.nist.gov/tracks/SM-KBP/2018/";
```

Comment:
The problem with using a common NIST_ROOT here is that the SeedlingOntology is sticking with 2018, but the LdcAnnotations is using 2019 in NistExamplesAndValidationTest.java, ScalingTest.java and maybe others.

Reply:
LDCAnnotations doesn't actually exist. It's simply a namespace. Let me know if you want the annotations derived from Seedling to be labeled as 2019

Reply:
Good point.  I'm fine with non-existent LDCAnnotations to reference 2019 instead of 2018 as long as all references to Seedling still point to 2018.

Reply:
Fixed